### ISSUE TITLE
We upgrade to the newer discord library that isn't discontinued

### ISSUE THREAD

**Issue by Nasa62** [FTL-Hyperspace/FTL-Hyperspace#22]
We could also upgrade to the newer discord library that isn't discontinued but this will require other changes in Hyperspace too, might be suited for a future PR although I myself don't care about Discord but maybe after this PR one could be stood up to change to the new Discord library in the build and the other HS devs could try to switch to it?

_Originally posted by @Nasa62 in https://github.com/FTL-Hyperspace/FTL-Hyperspace/pull/21#discussion_r896894866_

**Comment by Nasa62** [comment]
Note: Low priority and should be sometime after #21
